src-tauri/src/util.rs

```diff
@@ -136,12 +136,14 @@ pub fn check_ea_app_or_origin_running() -> bool {
 /// Checks if Northstar process is running
```

Comment:
Would also update the comment

```suggestion
/// Checks if Northstar process is running (ignores dedicated server)
```